packages/core/src/card/InteractableCard.css

```diff
@@ -1,50 +1,78 @@
 /* Styles applied to InteractableCard */
 .saltInteractableCard {
   box-shadow: var(--saltCard-boxShadow, var(--salt-overlayable-shadow));
-  border-width: var(--saltCard-borderWidth, var(--card-borderWidth));
+  border-width: var(--salt-size-border);
   border-style: var(--saltCard-borderStyle, var(--salt-container-borderStyle));
   border-color: var(--saltCard-borderColor, var(--salt-accent-borderColor));
   border-radius: var(--saltCard-borderRadius, 0);
   display: block;
+  position: relative;
   transition: box-shadow var(--salt-duration-instant) cubic-bezier(0.4, 0, 0.2, 1);
   padding: var(--saltCard-padding, var(--salt-spacing-300));
+  --card-accent-color: var(--saltCard-accentColor, var(--salt-accent-borderColor));
 }
 
 /* Styles applied to InteractableCard when variant="primary" */
 .saltInteractableCard-primary {
   background: var(--saltCard-background, var(--salt-container-primary-background));
+  border-color: var(--salt-container-primary-borderColor);
 }
 
 /* Styles applied to InteractableCard when variant="secondary" */
 .saltInteractableCard-secondary {
   background: var(--saltCard-background, var(--salt-container-secondary-background));
+  border-color: var(--salt-container-secondary-borderColor);
 }
 
 /* Styles applied to InteractableCard if `accentPlacement="bottom"` (default) */
-.saltInteractableCard-accentBottom {
-  --card-borderWidth: 0 0 var(--salt-size-accent) 0;
+.saltInteractableCard-accentBottom::after {
+  content: "";
```

Comment:
There's a lot of duplicated styling here. You should consider structuring this differently:

- Have a ::after selector that has the common styles with the only styles in placement applying positioning or visibility.